.golangci.example.yml

```diff
@@ -613,6 +613,13 @@ linters-settings:
       name: true
       begin: true
 
+  tenv:
+    # The following configurations enable checks prior to Go 1.17.
+    force: false
```

Comment:
The `force` option should be removed because it's just a way to enable/disable.
Then is just the same thing that adds or removes the linter.

Reply:
The force option is an option that determines whether to enable it for products older than Go1.7.
If force should be removed, what would be the best approach to implement the same?

Reply:
I think you don't have to handle the go1.17 constraint inside your linter.
And inside golangci-lint I don't think we have to handle that in this PR.

Reply:
Does this mean that we should run the linter for Go1.17, which does not provide t.Setenv?

Is it correct that the motivation for users of `tenv` is to replace `os.Setenv` with `t.Setenv`, so we don't need the force option and the go.mod version check?

Reply:
Your linter, by itself, doesn't have to care about the go version.
Inside golangci-lint, it's another topic, because a user has to choose to enable or disable a linter.
We can help by detecting the go version of the project but for me, it's outside of the scope of your PR.

Reply:
ok, I comprehended.
I'd remove force option and go version check.
thanks

> Inside golangci-lint, it's another topic, because a user has to choose to enable or disable a linter.

I agree with your opinion.